src/setup_env.sh

```diff
@@ -48,9 +48,12 @@ fi
 
 if [ ! -d $CURRENT_DIR/lib/OpenNMT-py ]; then
   echo "Cloning OpenNMT-py"
-  git clone https://github.com/chenzimin/OpenNMT-py.git $CURRENT_DIR/lib/OpenNMT-py
+  # git clone https://github.com/chenzimin/OpenNMT-py.git $CURRENT_DIR/lib/OpenNMT-py
+  git clone https://github.com/OpenNMT/OpenNMT-py.git $CURRENT_DIR/lib/OpenNMT-py
```

Comment:
Why is this change necessary? Because we have no guarantee that OpenNMT won't change it CLI and the options. Therefore it is better to clone one fixed version here

Reply:
I agree. Here it gets back to the original state.